client/api.go

```diff
@@ -71,7 +73,8 @@ type Client struct {
 }
 
 func NewClient(endpoint string) *Client {
-	timeout := time.Duration(10 * time.Second)
+	timeoutSeconds := viper.GetInt("CLIENT_TIMEOUT_SECONDS")
+	timeout := time.Duration(time.Duration(timeoutSeconds) * time.Second)
```

Comment:
```suggestion
	timeout := time.Duration(timeoutSeconds * time.Second)
```
Do we need that extra `time.Duration` cast? Looking at https://golang.org/pkg/time/#Duration

Reply:
Apparently it doesn't make a difference output-wise if you check out https://play.golang.org/p/XTY3Y8aUxgG - but probs for the best to do less casting

Reply:
The issue there is you cant multiply a `time.Duration` by an int. its gotta be multiplied by another `time.Duration`
<img width="687" alt="Screen Shot 2021-02-04 at 1 45 10 PM" src="https://user-images.githubusercontent.com/3021938/106940196-3e196280-66ef-11eb-8c90-ba01f7e6c983.png">